Philips.CodeAnalysis.MsTestAnalyzers/AssertMethodCallDiagnosticAnalyzer.cs

```diff
@@ -30,8 +29,7 @@ private void Analyze(SyntaxNodeAnalysisContext context)
 			{
 				foreach (Diagnostic diagnostic in Analyze(context, invocationExpression, memberAccessExpression) ?? Array.Empty<Diagnostic>())
 				{
-					IMethodSymbol memberSymbol = context.SemanticModel.GetSymbolInfo(memberAccessExpression).Symbol as IMethodSymbol;
-					if ((memberSymbol == null) || !memberSymbol.ToString().StartsWith("Microsoft.VisualStudio.TestTools.UnitTesting.Assert"))
+					if ((context.SemanticModel.GetSymbolInfo(memberAccessExpression).Symbol is not IMethodSymbol memberSymbol) || !memberSymbol.ToString().StartsWith("Microsoft.VisualStudio.TestTools.UnitTesting.Assert"))
```

Comment:
OK, seen this code snippet too often now. Consider putting this code into a Utility / Helper

Reply:
Beyond scope of this large already commit.